common/src/main/scala/com/gu/monitoring/CloudWatch.scala

```diff
@@ -0,0 +1,62 @@
+package com.gu.monitoring
+
+import java.util.concurrent.Future
+
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.regions.Regions.EU_WEST_1
+import com.amazonaws.services.cloudwatch.{AmazonCloudWatchAsync, AmazonCloudWatchAsyncClient, AmazonCloudWatchAsyncClientBuilder}
+import com.amazonaws.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest, PutMetricDataResult}
+import com.gu.aws.CredentialsProvider
+import com.gu.config.Configuration
+import com.gu.monitoring.CloudWatch.cloudwatch
+import com.typesafe.scalalogging.LazyLogging
+
+class CloudWatch(metrics: Seq[Dimension]) extends LazyLogging {
+
+  val application = "SupportWorkers"
+
+  //Common Dimensions
+  val commonDimensions: Seq[Dimension] =
```

Comment:
Since there is now just one common dimension, maybe this could become
`val stageDimension: Dimension = ...`